scripts/createReviewAppNamespace.js

```diff
@@ -0,0 +1,43 @@
+const CLOUDFLARE_API_URL = "https://api.cloudflare.com/client/v4"
+const CLOUDFLARE_ZONE_ID = "38e46d6be412f00b9fc0ef478f497ef1"
```

Comment:
Even though it's not sensitive, it bugs me to hard-code this. Can it come from the context instead?

Reply:
Updated